### PR TITLE
Fixes #9: Modified CreateTable.js

### LINKED ISSUE #9: Rename attribute U_ID in dynamodb schema
refrences #4

### PR DESCRIPTION
**Pull request by ayush0x00**
Fixes #9

### CODE REVIEW

File: DynamoDB/CreateTable.js [hunk 1]
```diff
@@ -13,25 +13,25 @@ var dynamodb = new AWS.DynamoDB();
...
+        ReadCapacityUnits: 10,
         WriteCapacityUnits: 10
     },
     GlobalSecondaryIndexes: [{
-        IndexName: "FIND_FILE",
+        IndexName: "FIND_FILE_BY_URL",
         KeySchema: [
             {
-                AttributeName: "U_ID",
+                AttributeName: "GS1_PK",
```
Comment (danishjamal104) [inline DynamoDB/CreateTable.js:34]: where is SK??

File: DynamoDB/CreateTable.js [hunk 2]
```diff
@@ -44,7 +44,7 @@ var params = {
         }
     }],
     LocalSecondaryIndexes: [{
-        IndexName: "FILTER_BY_NAME",
+        IndexName: "FIND_FILE_BY_NAME",
         KeySchema: [
             {
                 AttributeName: "PK",
```
Comment (danishjamal104) [inline DynamoDB/CreateTable.js:50]: For LSI we need to define only SK why have you define PK?

**Review by danishjamal104 — CHANGES_REQUESTED** [review]

File: DynamoDB/CreateTable.js (hunk 2 above)
Comment (ayush0x00) [reply DynamoDB/CreateTable.js:50]: We need to define both. PK value remains same as mentioned in the table.

Comment (lakshya-20) [reply DynamoDB/CreateTable.js:50]: Yes we need to mention both PK as well as SK while defining LSI "FIND_FILE_BY_NAME"

File: DynamoDB/CreateTable.js (hunk 1 above)
Comment (lakshya-20) [reply DynamoDB/CreateTable.js:34]: SK is the table's sort key.

Comment (lakshya-20) [reply DynamoDB/CreateTable.js:34]: For GSI "FIND_FILE_BY_URL" we don't need to define a composite index. Only PK can work for us.

Comment (lakshya-20) [reply DynamoDB/CreateTable.js:34]: For more information refer this [https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GSI.html](url)

Comment (lakshya-20) [reply DynamoDB/CreateTable.js:34]: For more information, you can refer to this  [https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GSI.html](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GSI.html)

**Review by lakshya-20 — APPROVED** [review]

File: DynamoDB/CreateTable.js (hunk 2 above)
Comment (danishjamal104) [reply DynamoDB/CreateTable.js:50]: so where is SK?

Comment (lakshya-20) [reply DynamoDB/CreateTable.js:50]: I think you are missing something.
"LS1_SK" is defined as the Sort Key for LSI "FIND_FILE_BY_NAME".

**Review by danishjamal104 — APPROVED** [review]
LGTM

**Pull request merged**